contactmps/templates/base.html

```diff
@@ -32,11 +32,14 @@
     <meta name="theme-color" content="#ffffff">
     {% endblock %}
 
+    <!-- this is not the right way, but I need bootstrap on index.html  -->
+    <link href="/static/stylesheets/app.css" rel="stylesheet" type="text/css">
```

Comment:
The site has two logically separate sets of pages which load different CSS and JS.

The first are the app pages -- the email creation and email sent pages -- which have specific CSS and JS. We currently build that into app.js and app.css.

The second is the landing page which is a container and doesn't need the CSS or JS for the app itself. We don't have support for this right now. I'll add it.